### PR TITLE
Added Chat Box Button

### PR DESCRIPTION
**Pull request by dsnehasish74**
Screenshot (282)

Issue:

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -147,6 +147,12 @@ $(document).ready(function () {
           classes: "buy-me-coffee-btn",
           text: "Buy me coffee",
         },
+        {
+          type: "chat",
+          label: "chat Box Button",
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:152]: Capitalize "chat" in label

File: src/components/icon/_chat.less [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.chat-btn {
```
Comment (shahednasser) [inline src/components/icon/_chat.less:1]: add `transition: all .2s`

File: src/components/icon/_chat.less [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    font-family: "Font Awesome 5 Free";
+    display: inline-block;
+    font-style: normal;
+    font-variant: normal;
+    text-rendering: auto;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  &:hover {
+    box-shadow: 0px 1px 7px #4d4d4d;
```
Comment (shahednasser) [inline src/components/icon/_chat.less:13]: change box-shadow color to `#838383`

**Review by shahednasser — CHANGES_REQUESTED** [review]
It all looks good, I just need you to make some changes requested below and also please check off the idea in the "button ideas" file

**Comment by dsnehasish74** [comment]
All the changes are done ............... please kindly check this

**Comment by shahednasser** [comment]
Great thank you!

**Pull request merged**